build.xml

```diff
@@ -843,6 +843,7 @@ TODO:
 
     <path id="quick.bin.tool.path">
       <path refid="quick.repl.build.path"/>
+      <path refid="quick.repl-jline.build.path"/>
```

Comment:
this should not be necessary since the quick.repl build step copies all repl-jline + shaded jline classes into build/quick/classes/repl